sceval_frontend/src/controllers/ModeAPI.ts

```diff
@@ -42,6 +40,30 @@ export interface ErrorResponse {
   status: number;
 }
 
+export class MODECONSTANTS {
+  public static MODE_API_BASE_URL: string = 'https://api.tinkermode.com/';
```

Comment:
It can be rewrited using only `static`. It's better using also `readonly` modifier.

Reply:
I think it's also use `module` and `export const` for this like as below.

```
module Constants {
    export const SOME_VALUE: string = "foo";
}
```